src/app/tests/suites/TV_ApplicationBasicCluster.yaml

```diff
@@ -58,16 +56,33 @@ tests:
       response:
           value: 1
 
-    # TODO: Support chars validation
-    - label: "Read attribute id"
-      disabled: true
+    - label: "Read attribute application status"
       command: "readAttribute"
-      attribute: "application id"
+      attribute: "application status"
       response:
-          value: "appId"
+          value: 0
+    # TODO: Enable once attribute struct is supported
+    # - label: "Read attribute application status"
+    #   command: "readAttribute"
+    #   attribute: "application app"
+    #   response:
+    #       values:
+    #           - name: "arg1"
+    #             value:
+    #                 {
+    #                     catalogVendorId: 123,
+    #                     applicationId: "applicationId"
+    #                 }
 
-    - label: "Read attribute catalog vendor id"
+    # TODO: Support chars validation
+    - label: "Read attribute application version"
       command: "readAttribute"
-      attribute: "catalog vendor id"
+      attribute: "application version"
       response:
-          value: 1
+          value: "exampleVersion"
+    # - label: "Read attribute application allowed vendor list"
```

Comment:
Uncomment.  If it's marked disabled it will be ignored anyway.

Reply:
Updated.